### DISCUSSION TITLE
How to debug SSR network traffic?

### DISCUSSION THREAD

**Discussion by stefan-girlich** [vercel/next.js#43861] [Help]
My web app is fetching data via HTTP API calls in `getServerSideProps`.
I'd like to debug these HTTP calls via Chrome DevTools as I'd do if they were client-side calls. 

I set up SSR debugging [as described in the docs](https://nextjs.org/docs/advanced-features/debugging#server-side-code), but there is no _Network_ tab as usual:
<img width="1912" alt="image" src="https://user-images.githubusercontent.com/1755315/206552228-cb22cce8-c454-4854-9d55-a8aab059489d.png">

What's the designated way to debug SSR network calls in Next.js?

**Comment by chaitanyaj2222** [comment]
Hey @stefan-girlich . I stumbled upon the same issue. I searched the internet but I couldn't find a way. Additionally, the SSR setup described in the docs doesn't `console.log` either

Were you able to figure out a solution to this?
